src/video_core/renderer_vulkan/vk_pipeline_cache.cpp

```diff
@@ -59,7 +63,17 @@ const ComputePipeline* PipelineCache::GetComputePipeline() {
     return pipeline;
 }
 
-void PipelineCache::RefreshGraphicsKey() {
+bool ShouldSkipShader(u64 shader_hash, const char* shader_type) {
+    static constexpr u64 skip_hashes[] = {0xdeadbeefULL};
```

Comment:
array cant be empty or it wont compile, and i figured it would be helpful as an indication of what to put. if there is a better way please let me know!

Reply:
Maybe just comment the lines and say to uncomment if needed?

Reply:
it was suggested in the discord to tweak how the array was checked to allow for an empty array and is reflected in latest commit